front/src/modules/study/components/StudyCard.tsx

```diff
@@ -8,9 +8,10 @@ import studyLogo from 'assets/pictures/views/study.svg';
 import { budgetFormat } from 'utils/numbers';
 import { dateTimeFormatting } from 'utils/date';
 import { useDispatch } from 'react-redux';
-import { updateScenarioID, updateStudyID } from 'reducers/osrdconf';
 import { RiFolderChartLine } from 'react-icons/ri';
 import { StudyWithScenarios } from 'common/api/osrdEditoastApi';
+import { useOsrdActions } from 'common/osrdContext';
+import { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Comment:
Could you add `type` to all other type imports of the file please :)
```suggestion
import type { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
```

Reply:
Done